src/components/StarRating/src/StarRatingSelector.vue

```diff
@@ -0,0 +1,62 @@
+<template>
```

Comment:
Rather than creating a StarRatingSelector component, we could instead an `editable` boolean prop in StarRating.vue. I think removing this one layer would reduce the amount of code and make things a bit easier to follow.